Snowplow/Internal/Tracker/SPTracker.h

```diff
@@ -89,7 +89,7 @@ NS_SWIFT_NAME(Tracker.Builder)
 
  @param subject An associated subject
  */
-- (void) setSubject:(SPSubject *)subject;
+- (void) setSubject:(nullable SPSubject *)subject;
```

Comment:
It isn't a change actually. On SPTracker I've introduced `NS_ASSUME_NONNULL_BEGIN` (line 65). It impose that all the properties and arguments are `nonnull`. So I needed to set some of them `nullable` in order to be coherent with the legacy behaviour. `setSubject` was considered optional and nullable and in the version 2.0 it would be considered just deprecated 1.0 API, hence the `nullable` attribute as I don't change the it always worked.